spec/TestSpec.ts

```diff
@@ -0,0 +1,23 @@
+import { getTextNodeAccessibleName } from '../src/accessible-name';
```

Comment:
Please name .ts files in snake_case for better Closure compatibility http://go/tsstyle#file-names.

I also suggest having a _test.ts file for each normal file, so `accessible_name.ts` and `accessible_name_test.ts`.

Reply:
Also http://go/tsstyle#code-location suggests putting `_test` files in the same directory as the code under test.

Reply:
Will do!